pulp_2to3_migration/app/plugin/content.py

```diff
@@ -198,62 +202,93 @@ def get_remote_by_importer_id(importer_id):
                 return
             return pulp2importer.pulp3_remote
 
+        futures = []
+        batch_size = 500
+        lazy_type = content_type in migrator.lazy_types
+        artifactless_type = content_type in migrator.artifactless_types
+        has_future = content_type in migrator.future_types
```

Comment:
:+1:

Reply:
I really like it, it looks clean. Is it worth adding a verb, so it's clear that these are booleans and not types themselves?
`is_lazy_type` ?
`is_artifactless_type` or `has_no_artifact`? 
`has_future` is perfect

Reply:
I can rename them no problem, however when I read the code `if lazy_type` i have no issues understanding it. I guess it is matter of personal preference.

Reply:
I think it's more of the convention to signify that the boolean is there and not any other value.